Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/wrapper/impl/ConceptEntryWrapperCreator.java

```diff
@@ -79,6 +86,15 @@
 
                 }
             }
+            
+          //For getting the Review Request of the entry
+            ReviewRequest revReq = commentsManager.getEntry(entry.getId());
```

Comment:
let's write out the variable name. there is really no good reason for abbreviations here.